eng/build.sh

```diff
@@ -13,9 +13,7 @@ usage()
   echo "  --binaryLog                Create MSBuild binary log (short: -bl)"
   echo ""
   echo "Actions:"
-  echo "  --bootstrap                Force the build of the bootstrap compiler"
```

Comment:
Why is this being removed? How do we force a bootstrap build now?

Reply:
Well ... as you spotted earlier we always build the bootstrapper so bootstrap build currently happens always when using build.sh, but yes we should have this switch and the short circuit when a proto exists.